src/main/java/pl/wedding/management/service/reservation/domain/converter/ReservationConverter.java

```diff
@@ -0,0 +1,23 @@
+package pl.wedding.management.service.reservation.domain.converter;
+
+import lombok.NoArgsConstructor;
+import pl.wedding.management.service.converter.Converter;
+import pl.wedding.management.service.reservation.domain.entity.ReservationEntity;
+import pl.wedding.management.service.reservation.domain.entity.ReservationDto;
+
+import java.time.temporal.ChronoUnit;
+
+@NoArgsConstructor
+public class ReservationConverter implements Converter<ReservationDto, ReservationEntity> {
+
+    @Override
+    public ReservationDto convertFromEntity(ReservationEntity reservationEntity) {
+        return new ReservationDto(reservationEntity.getId(),
+                reservationEntity.getReservationStart().truncatedTo(ChronoUnit.MINUTES),
+                reservationEntity.getReservationEnd().truncatedTo(ChronoUnit.MINUTES),
```

Comment:
not-npe-safe !!

Reply:
Czy taka adnotacja jest wystarczająca?

Reply:
Nie, taka adnotacja nie jest wystarczająca. Ona mówi, że przesłany obiekt nie może być nullem.
Augustyn chciał powiedzieć, że w momencie kiedy `reservationEntity.getReservationStart()` zwróci nam nulla to zawołanie na niej jakiejkolwiek metody będzie skutkowało NPE.  Żeby to naprawić proponuję najpierw napisać Ci test, gdzie spróbujesz uruchomić tą metodę z ReservationEntity który będzie miał w każdym polu wartość NULL. Jak odpalisz test, powinien Ci się wysypać. Wtedy napraw logikę aplikacji, odpal test jeszcze raz. Jeśli będzie zielony - super.